R/utilities-data-set.R

```diff
@@ -134,17 +135,28 @@ dataSetToDataFrame <- function(dataSets) {
 #'
 #' dataSets <- loadDataSetsFromExcel(
 #'   xlsFilePath = xlsFilePath,
-#'   importerConfiguration = importerConfiguration,
+#'   importerConfigurationOrPath = importerConfiguration,
 #'   importAllSheets = FALSE
 #' )
-loadDataSetsFromExcel <- function(xlsFilePath, importerConfiguration, importAllSheets = FALSE) {
+#'
+#' importerConfigurationFilePath <- system.file("extdata", "dataImporterConfiguration.xml", package = "ospsuite")
+#'
+#' dataSets <- loadDataSetsFromExcel(
+#'   xlsFilePath = xlsFilePath,
+#'   importerConfigurationOrPath = importerConfigurationFilePath,
+#'   importAllSheets = FALSE
+#' )
+loadDataSetsFromExcel <- function(xlsFilePath, importerConfigurationOrPath, importAllSheets = FALSE) {
   validateIsString(xlsFilePath)
-  validateIsOfType(importerConfiguration, "DataImporterConfiguration")
+  if (is.character(importerConfigurationOrPath)) {
```

Comment:
I am beign a bit picky here but I would suggest to do like so
importerConfiguration<-importerConfigurationOrPath;
if (is.character(importerConfigurationOrPath)) {
  importerConfiguration <- loadDataImporterConfiguration(importerConfigurationOrPath)
}

so that after this , you KNOW That you have a importerConfiguration

and you not using `importerConfigurationOrPath$ref` which would not work if it was a string